.github/workflows/build-and-deploy-to-ck8s.yaml

```
@@ -63,11 +61,15 @@ jobs:
       run: |-
         echo $DOCKER_PASSWORD \
           | docker login --username $DOCKER_USER --password-stdin harbor.$DOMAIN
+    
+    - name: Set container image tag
+      run: |-
+        echo "TAG=${{ github.ref_name }}-$(git describe --tags --always)" >> $GITHUB_ENV
```

I'd rather use [environment variables](https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables) as opposed to [contexts](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context) in shell snippets. Makes the whole thing more reusable across CI/CD solutions.

```suggestion
        echo "TAG=${GITHUB_REF_NAME}-$(git describe --tags --always)" >> $GITHUB_ENV
```

---

Have removed this line and gone with the other approach you suggested instead!